pheget/frontend/templates/frontend/region.html

```diff
@@ -14,128 +14,160 @@
         <div id="home-searchbox">
           <form id="variant-search">
             <div class="input-group">
+              <div class="input-group-prepend">
+                  <a href="/" class="btn btn-secondary btn-sm" role="button" aria-pressed="true"><span class="fa fa-home"></span></a>
+              </div>
               <input id="query-field" name="query-field" autocomplete="off" class="form-control"
                      type="text"
                      placeholder="Search for a variant, e.g. chr19:488506 or rs10424907" autofocus/>
               <div class="input-group-append">
-                <button id="navigate-variant" class="btn btn-primary" type="submit">Search <span
+                <button id="navigate-variant" class="btn btn-secondary" type="submit"><span
                     class="fa fa-search"></span></button>
               </div>
             </div>
           </form>
         </div>
       </div>
     </div>
-    <center>
-      <a href="/" class="btn btn-primary btn-sm" role="button" aria-pressed="true"><span class="fa fa-home"></span> Back
-        to PheGET Homepage</a>
-    </center>
+    
     <br>
     {#  FIXME: This label does not update as the plot is panned or zoomed. We can resolve this in the future when we move to a dynamic frontend #}
-    <h1 style="margin-top: 1em;"><strong>Single-tissue eQTLs near {{ chrom }}:{{ "{:,}".format(center) }} </strong></h1>
+    <h1 style="margin-top: 1em;"><strong>Single-tissue eQTLs in {{ chrom }}:{{ "{:,}".format(start) }}-{{ "{:,}".format(end) }} </strong></h1>
+  </div>
+
+  <div class="container-fluid">
+    <div class="row">
+      <div id="lz-plot" class="ten columns">
```

Comment:
A small aside: "ten columns" is a skeleton.css construct; this page started life copied from a locuszoom example. It has no effect in bootstrap, which is what we are using for layout now. (I've pushed a change to remove that; feel free to clean up any other usages you might see)